boms/legacy-ee/pom.xml

```diff
@@ -243,12 +243,12 @@
             </dependency>
             <dependency>
                 <groupId>${full.maven.groupId}</groupId>
-                <artifactId>wildfly-opentelemetry-legacy-namespace</artifactId>
+                <artifactId>wildfly-opentelemetry</artifactId>
                 <version>${full.maven.version}</version>
             </dependency>
             <dependency>
                 <groupId>${full.maven.groupId}</groupId>
-                <artifactId>wildfly-opentelemetry-api-legacy-namespace</artifactId>
+                <artifactId>wildfly-opentelemetry-api</artifactId>
                 <version>${full.maven.version}</version>
             </dependency>
```

Comment:
Both of these should just be deleted. Anything that needs the two non-legacy artifacts shouldn't be relying on this bom to manage that dep. And I don't think anything is doing that.